examples/templates/digitalocean-linux/main.tf

```diff
@@ -9,27 +9,26 @@ terraform {
   }
 }
 
-provider "coder" {
-}
+provider "coder" {}
 
-variable "step1_do_project_id" {
+variable "do_project_id" {
```

Comment:
what are your thoughts on dropping `do_` prefix from all variable names?

Reply:
I also thought about doing that but wasn't sure, we should do it.
`step1_do_project_id` would become `project_id` then?

Reply:
yes